ci/tests/test_01_functional/test_04_index_ci/__init__.py

```diff
@@ -134,7 +135,119 @@ def test_04_indexci_runs_specific_value_validators(self):
         s, summary = Engine(
             index_location=mock_loc,
             the_state=st,
-            schema_validators=None,
+            skip_schema=True,
             value_validators=["GitCloneValidator"]
         ).run()
         self.assertFalse(s)
+
+    def test_05_indexci_run_from_command_line(self):
+        _, mock_loc = self.setup_mock_location(
+            {
+                "bamachrn.yaml": {
+                    FieldKeys.PROJECTS: [
+                        {
+                            FieldKeys.ID: 1,
+                            FieldKeys.APP_ID: "bamachrn",
+                            FieldKeys.JOB_ID: "python",
+                            FieldKeys.DESIRED_TAG: "latest",
+                            FieldKeys.GIT_URL: "https://github.com/bamachrn/ccc"
+                                               "p-python",
+                            FieldKeys.GIT_PATH: "demo",
+                            FieldKeys.GIT_BRANCH: "master",
+                            FieldKeys.TARGET_FILE: "Dockerfile.demo",
+                            FieldKeys.NOTIFY_EMAIL: "hello@example.com",
+                            FieldKeys.BUILD_CONTEXT: "./",
+                            FieldKeys.DEPENDS_ON: "centos/centos:latest"
+                        }
+                    ]
+                }
+            }
+        )
+        cmd = [
+            "python",
+            str(path.join(
+                path.dirname(path.realpath(__file__)),
+                "..",
+                "..",
+                "..",
+                "container_index",
+                "run.py",
+            )),
+            "-i",
+            mock_loc
+        ]
+
+        check_call(cmd)
+
+    def test_06_indexci_override_schema_validators(self):
+        _, mock_loc = self.setup_mock_location(
+            {
+                "bamachrn.yaml": {
+                    FieldKeys.PROJECTS: [
+                        {
+                            FieldKeys.ID: 1,
+                            FieldKeys.APP_ID: "bamachrn",
+                        }
+                    ]
+                }
+            }
+        )
+        cmd = [
+            "python",
+            str(path.join(
+                path.dirname(path.realpath(__file__)),
+                "..",
+                "..",
+                "..",
```

Comment:
Can you think of a way to avoid this ?

Reply:
I will definitely try to figure out a way to avoid this.
But for now, as interim, we should be ok?